Zend/zend_fibers.h

```diff
@@ -92,9 +92,16 @@ static const zend_uchar ZEND_FIBER_STATUS_BAILOUT   = 0x20;
 
 static const zend_uchar ZEND_FIBER_STATUS_FINISHED  = 0x2c;
 
+/* These functions create and manipulate a Fiber object, allowing any internal function to start, resume, or suspend a fiber. */
+ZEND_API zend_fiber *zend_fiber_create(const zend_fcall_info *fci, const zend_fcall_info_cache *fci_cache);
+ZEND_API void zend_fiber_start(zend_fiber *fiber, zval *params, uint32_t param_count, zend_array *named_params, INTERNAL_FUNCTION_PARAMETERS);
+ZEND_API void zend_fiber_suspend(zval *value, INTERNAL_FUNCTION_PARAMETERS);
```

Comment:
This doesn't fit, it should accept the fiber as first param.

Reply:
I think this is right, suspend current fiber does not need fiber param, just as `Fiber::suspend()` is a static function, it was also like `sche_yield()` does not need PID :)

Reply:
I think from an API point of view, it's better to accept the zend_fiber and make assertions that it's being used correctly ... but it's not really required, just a nicer API.

Reply:
The only correct zend_fiber to pass here is EG(current_fiber), so requiring it to be passed seems unnecessary.